docs/source/user_guide/installation/install-from-source.rst

```diff
@@ -34,25 +34,42 @@ To install PyBaMM, you will need:
 
 You can install the above with
 
-.. tab:: Ubuntu
+.. tab:: Ubuntu/Debian/Linux Mint
 
 	.. code:: bash
 
-		sudo apt install python3.X python3.X-dev libopenblas-dev gcc gfortran graphviz
+		apt install python3.X python3.X-dev libopenblas-dev gcc gfortran graphviz
 
 	Where ``X`` is the version sub-number.
 
-	.. note::
+.. tab:: Fedora/CentOS
 
-		On Windows, you can install ``graphviz`` using the `Chocolatey <https://chocolatey.org/>`_ package manager, or
-		follow the instructions on the `graphviz website <https://graphviz.org/download/>`_.
+	.. code:: bash
+
+		dnf install python3.X-devel openblas-devel gcc gcc-fortran graphviz
+
+	Where ``X`` is the version sub-number.
+
+.. tab:: Arch Linux
+
+	.. code:: bash
+
+		pacman -S python openblas gcc gcc-fortran graphviz
```

Comment:
```suggestion
		sudo pacman -S python openblas gcc gcc-gfortran graphviz
```

I'm not sure if it is `gcc-fortran` or `gcc-gfortran` – please confirm

Reply:
I think the sudo in the suggestion should be added. This probably should not have been resolved yet

Reply:
```suggestion
		sudo pacman -S python openblas gcc gcc-fortran graphviz
```